signac/contrib/indexing.py

```diff
@@ -159,7 +159,7 @@ class RegexFileCrawler(BaseCrawler):
         MyCrawler.define('.*\/a_(?P<a>\d+)\.txt', 'TextFile')
     """
     "Mapping of compiled regex objects and associated formats."
-    definitions = dict()
+    definitions = dict()  # type: ignore
```

Comment:
Explanation: This requests a type annotation for the `dict`. I added ignores to all deprecated modules where necessary, without questioning it.